src/p2p/behaviour.rs

```diff
@@ -15,13 +15,15 @@ use libp2p::kad::{Kademlia, KademliaConfig, KademliaEvent, Quorum};
 // use libp2p::mdns::{MdnsEvent, TokioMdns};
 use libp2p::ping::{Ping, PingEvent};
 // use libp2p::swarm::toggle::Toggle;
+use libp2p::floodsub::FloodsubEvent;
 use libp2p::swarm::{NetworkBehaviour, NetworkBehaviourEventProcess};
 use multibase::Base;
 use std::{convert::TryInto, sync::Arc};
 use tokio::task;
 
 /// Behaviour type.
 #[derive(libp2p::NetworkBehaviour)]
+#[behaviour(out_event = "BehaviourEvent", event_process = true)]
```

Comment:
```suggestion
#[behaviour(event_process = true)]
```

Why is the `OutEvent` needed here? Given that you set `event_process = true` there is no need for it, right?

Reply:
Resolved!